tests/cypress/integration/actions_tasks/continue_frame_n.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Paste labels from one task to another.', { browser: '!firefox' }, () => {
+    const task = {
+        name: 'Test "Continue frame N"',
+        label: 'Test label',
+        attrName: 'Test attribute',
+        attrValue: 'Test attribute value',
+    };
+
+    const imagesCount = 3;
+    const imageFileName = `image_${task.name.replace(' ', '_').toLowerCase()}`;
+    const width = 100;
+    const height = 100;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+
+    before(() => {
+        cy.visit('/');
+        cy.login();
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, task.name, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.goToTaskList();
+        cy.createAnnotationTask(task.name, task.label, task.attrName, task.attrValue, archiveName);
+    });
+
+    after(() => {
+        cy.logout();
+        cy.getAuthKey().then((authKey) => {
+            cy.deleteTasks(authKey, [task.name, task.nameSecond]);
+        });
+    });
+
+    describe('Test "Continue frame N"', () => {
+        it('Open job, go to the 3rd frame, close task, reopen, notification exists.', () => {
+            cy.openTaskJob(task.name);
+            cy.get('.cvat-player-frame-selector')
```

Comment:
You can use `cy.checkFrameNum` for simplification

Reply:
Sure, didn't know about this command